shared/src/business/entities/cases/Case.ts

```diff
@@ -1785,20 +1787,14 @@ export class Case extends JoiValidationEntity {
     return this;
   }
 
-  generateTrialSortTags(): {
-    hybrid: string;
-    nonHybrid: string;
-  } {
-    return generateTrialSortTags(this);
-  }
-
   /**
    * set as calendared
    * @param {object} trialSessionEntity - the trial session that is associated with the case
    * @returns {Case} the updated case entity
    */
   setAsCalendared(trialSessionEntity) {
     this.updateTrialSessionInformation(trialSessionEntity);
+    this.manuallyAddedToTrial = true;
```

Comment:
Won't this also be set from `setTrialSessionCalendarInteractor` for eligible, non-manually added cases?

Reply:
Dang it

Reply:
Not a _problem_ per se, but misleading

Reply:
`addedToTrial` it is